kubric/core/objects.py

```diff
@@ -130,13 +134,42 @@ def _valid_bounds(self, proposal):
         raise tl.TraitError(f"lower bound cannot be larger than upper bound ({lower} !<= {upper})")
     return lower, upper
 
+  @property
+  def bbox_3d(self):
+    """ 3D bounding box as an array of 8 edges shape = [8, 3]."""
```

Comment:
what's an "edge shape"?
Given [8,3] I would assume these are the 8 corners of the bbox? (But might be mistaken)